src/command/backup/common.h

```diff
@@ -14,6 +14,8 @@ Common Functions and Definitions for Backup and Expire Commands
 Backup constants
 ***********************************************************************************************************************************/
 #define BACKUP_PATH_HISTORY                                         "backup.history"
+#define DATE_TIME_REGEX                                             "[0-9]{8}\\-[0-9]{6}"
```

Comment:
Maybe add a little comment with example of expected datetime and it's format (%Y%m%d-%H%M%S)?

Reply:
done